Product/component/PGallery.jsx

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
-import styles from './PGallery.module.css'
+import styles from './pGallery.module.css'
```

Comment:
```suggestion
import styles from './PGallery.module.css'
```
Typo here giving errors